lighthouse-core/lib/manifest-parser.js

```diff
@@ -425,7 +425,7 @@ function parse(string, manifestUrl, documentUrl) {
     return {
       raw: string,
       value: undefined,
-      debugString: 'ERROR: file isn\'t valid JSON: ' + e,
+      warning: 'ERROR: file isn\'t valid JSON: ' + e,
```

Comment:
I think @paulirish wanted this to be `errorMesssage`, as this was the one true place it can fail (everywhere else it falls back to a default)

Reply:
(it will be interesting to see if you change it just here (and `errorMessage: undefined` below) if tsc will correctly pick up all the places it needs to be adjusted automatically since the artifact is dynamically typed from the inferred return value of this function...)

Reply:
Oh meant to reply to this, this was an intentional feigned ignorance about that part of the comment ;)

I don't think manifest-parser should to try to name its properties based on how one audit will pass/fail because of its `value`. From the perspective of manifest-parser all of these strings are messages explaining that something was abnormal about the raw value (there are other cases that we ignore the string and provide a default where it's a warning).

Since we're taking the stance that these are all warnings and not total failures, seems like we should lean in to it :)

Reply:
on the one hand.. this field represents why the manifest parser was unable to parse.
but on the other hand.. i dont really care. 😑 i'm fine with `warning` if you'd prefer it.

Reply:
lol, I'd prefer it given the agreement we have around everything else in there labelled `ERROR:` being a warning :)

sg!

Reply:
> this field represents why the manifest parser was unable to parse.

I also agree with this. It's the only real failure for a manifest. Everything else is a warning that something was malformed but the browser will fall back to a default, so the manifest will still work.

But I also don't care that much, it's just (minor) ergonomics of the result, not correctness.